cast_test.go

```diff
@@ -14,6 +14,231 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestToUintE(t *testing.T) {
+	tests := []struct {
+		input  interface{}
+		expect uint
+		iserr  bool
+	}{
+		{int(8), 8, false},
```

Comment:
Uint tests without any negative values ... I guess that should have returned an error.